hera_qm/xrfi.py

```diff
@@ -222,9 +251,52 @@ def detrend_medfilt(d, Kt=8, Kf=8):
     return f[Kt:-Kt, Kf:-Kf]
 
 
+def detrend_meanfilt(d, flags=None, Kt=8, Kf=8):
```

Comment:
This is our first filtering function that can actually use the flags to mask known outliers. I want to build on this to make a median filter that can do the same, but I couldn't find a suitable out-of-box solution so I started with this.